manifests/h/HTTPToolKit/HTTPToolKit/1.4.1/HTTPToolKit.HTTPToolKit.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate
+
+PackageIdentifier: HTTPToolKit.HTTPToolKit
+PackageVersion: 1.4.1
+PackageLocale: en-US
+Publisher: HTTP Toolkit
+PackageName: HTTP Toolkit
+PackageUrl: https://httptoolkit.tech/
+License: Copyright (C) 2021 HTTP Toolkit
+LicenseUrl: https://tldrlegal.com/l/agpl3
+ShortDescription: Beautiful & open-source tools to debug, test and develop with HTTP(S).
+Moniker: HTTPToolkit

```

Comment:
```suggestion
Moniker: httptoolkit
```

Reply:
Quick question, is `Moniker` case-sensivite for winget install/list/upgrade? In below comment, I mentioned 1.3.0 pull request, so if it does not affect, we can keep them with the original format. Forcing them to lowercase looks good too 😄 .

Reply:
I did a test on my device, it's seems like it's not case-sensitive, but it's better to update it lowercase because most monikers are in lower-case.